scripts/webpack.conf.js

```diff
@@ -70,6 +70,13 @@ module.exports = Promise.all([
         entry: './src/injected',
       },
     },
+  }, (config) => {
+    config.plugins.push(
+      new WrapperWebpackPlugin({
+        header: `if(window[Symbol.for('${INIT_FUNC_NAME}')]!==1)`,
+        afterOptimization: true,
+      }),
```

Comment:
Why is the condition added this way instead of writing in the source code?

Reply:
This will prevent webpack bootstrap code from running in a potentially hacked environment.

Reply:
I'd like to do the same in `browser.js` but I didn't know how to do it easily...

Reply:
Yay, I found a way, see the [`fixup!` commit](https://github.com/violentmonkey/violentmonkey/pull/842/commits/1edc0ade7208971bb2d381ceebc33406c215847a): added `test: /^browser\.js$/` in the main config.

Reply:
It seems that `afterOptimization: true` is not necessary?

Reply:
You're right. It's a leftover form my experiments.